{{cookiecutter.project_slug}}/CONTRIBUTING.rst

```diff
@@ -76,56 +95,68 @@ Ready to contribute? Here's how to set up ``{{ cookiecutter.project_slug }}`` fo
+
+#. To ensure a consistent coding style, install the ``pre-commit`` hooks to your local clone::
 
     $ pre-commit install
 
-   Special style and formatting checks will be run when you commit your changes. You
-   can always run the hooks on their own with:
+  On commit, ``pre-commit`` will check that{% if cookiecutter.use_black == 'y' %} ``black``, ``blackdoc``, ``isort``,{% endif %} ``flake8``, and ``ruff`` checks are passing, perform automatic fixes if possible, and warn of violations that require intervention. If your commit fails the checks initially, simply fix the errors and re-commit.
+
+  You can also run the hooks manually with::
 
     $ pre-commit run -a
 
+  If you want to skip the ``pre-commit`` hooks temporarily, you can pass the ``--no-verify`` flag to `$ git commit`.
+
 #. Create a branch for local development::
 
     $ git checkout -b name-of-your-bugfix-or-feature
 
-   Now you can make your changes locally.
+  Now you can make your changes locally.
 
-#. When you're done making changes, check that your changes pass ``black``, ``blackdoc``, ``flake8``, ``isort``, ``ruff``, and the tests, including testing other Python versions with tox::
+#. When you're done making changes, we **strongly** suggest running the tests in your environment or with the help of ``tox``::
 
-    $ black --check {{ cookiecutter.project_slug }} tests
-    $ isort --check {{ cookiecutter.project_slug }} tests
-    $ ruff {{ cookiecutter.project_slug }} tests
-    $ flake8 {{ cookiecutter.project_slug }} tests
-    $ blackdoc --check {{ cookiecutter.project_slug }} docs
     $ python -m pytest
+    # Or, to run multiple build tests
     $ tox
 
-   To get ``black``, ``blackdoc``, ``flake8``, ``isort``, ``ruff``, and tox, just pip install them into your virtualenv.
-
 #. Commit your changes and push your branch to GitHub::
 
     $ git add .
     $ git commit -m "Your detailed description of your changes."
     $ git push origin name-of-your-bugfix-or-feature
 
-#. If you are editing the docs, compile and open them with::
+  If ``pre-commit`` hooks fail, try re-committing your changes (or, if need be, you can skip them with `$ git commit --no-verify`).
 
+#. Submit a `Pull Request <https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request>`_ through the GitHub website.
+
+#. When pushing your changes to your branch on GitHub, the documentation will automatically be tested to reflect the changes in your Pull Request. This build process can take several minutes at times. If you are actively making changes that affect the documentation and wish to save time, you can compile and test your changes beforehand locally with::
+
+    # To generate the html and open it in your browser
     $ make docs
-    # or to simply generate the html
-    $ cd docs/
-    $ make html
+    # To uniquely generate the html
```

Comment:
Does `uniquely` work in that context? (says the Francophone to the Anglo 😅)
```suggestion
    # To only generate the html
```

Reply:
It's technically correct, but I think you're right that it's nearly a `francisisme` in English :laughing:

Reply:
We're starting to corrupt you!